ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/sbom/index.tsx

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { createFileRoute } from '@tanstack/react-router';
+
+import { prefetchUseRepositoriesServiceGetOrtRunByIndex } from '@/api/queries/prefetch';
+import { useRepositoriesServiceGetOrtRunByIndexSuspense } from '@/api/queries/suspense';
+import { OpenAPI } from '@/api/requests';
+import CycloneDXDark from '@/assets/cyclonedx-logo-black.svg';
+import CycloneDXLight from '@/assets/cyclonedx-logo-white.svg';
+import SPDX from '@/assets/spdx-logo-color.svg';
+import { LoadingIndicator } from '@/components/loading-indicator';
+import { useTheme } from '@/components/theme-provider-context';
+import { ToastError } from '@/components/toast-error';
+import { Button } from '@/components/ui/button';
+import {
+  Card,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+} from '@/components/ui/card';
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from '@/components/ui/tooltip';
+import { toast } from '@/lib/toast';
+
+const SBOMComponent = () => {
+  const params = Route.useParams();
+  // For CycloneDX, no SVG logo was found which would scale automatically
```

Comment:
I think "scale" is the wrong word here, as it's not related to sizing. "Adapt to the color theme" or so would probably be more fitting.